BatalanVlad/src/PaooGame/Game.java

```diff
@@ -134,7 +170,7 @@ public synchronized void StartGame()
      */
     public synchronized void StopGame()
     {
-        if(runState == true)
+        if(runState)
         {
                 /// Actualizare stare thread
             runState = false;
```

Comment:
Don't over comment. If you need comments to explain something that is not a class/public function you have one of the following situations:
1) that piece of code is hard and it needs clarifications
2) your code is not expressive enough, and you need refactorization

If you are not in 1) then you don't need to comment that stuff. Keep that in mind.

Reply:
Made the simplifications